src/options/view.rs

```diff
@@ -330,9 +330,25 @@ impl TimeFormat {
             "iso" => Ok(Self::ISOFormat),
             "long-iso" => Ok(Self::LongISO),
             "full-iso" => Ok(Self::FullISO),
-            fmt if fmt.starts_with('+') => Ok(Self::Custom {
-                fmt: fmt[1..].to_owned(),
-            }),
+            fmt if fmt.starts_with('+') => {
+                if fmt.contains('\n') {
+                    let mut lines = fmt[1..].lines();
+                    Ok(Self::Custom {
+                        non_recent: lines.next().expect("First line should exist.").to_owned(),
+                        recent: Some(
+                            lines
+                                .next()
+                                .expect("Second line should exist since fmt contains `\\n`.")
+                                .to_owned(),
+                        ),
```

Comment:
Those error messages are not meaningful enough, could you do something such as `Format should be:` and then specify based on the case ?

Reply:
This is not doing chrono syntax validation, ~~the two `.expect` calls here are expected to never fail, they are just `.unwrap` with sanity check messages~~.

I just realized they will fail if I only supply a plus sign `+`, will fix that in a moment.

Reply:
Should empty format string be allowed?  It would be valid chrono syntax, and a potential usage is to "only show time for recent (or non-recent) files".

Reply:
How is this:  
https://github.com/eza-community/eza/blob/78d26098d28dc24d0753a51826b86823e452f109/src/options/view.rs#L333-L365